gravity.sh

```diff
@@ -541,15 +541,15 @@ gravity_DownloadBlocklists() {
     # it (in case it doesn't exist)
     # First, check if the directory is writable
     directory="$(dirname -- "${saveLocation}")"
-    directory_permissions=$(stat -c %a ${directory})
+    directory_permissions=$(stat -c %a ${directory} 2>/dev/null)
     if [ $directory_permissions -lt 700 ]; then
```

Comment:
I'm not sure why there's a variable created for a single use. 

```suggestion
    if [ $(stat -c %a ${directory} -lt 700) ]; then
```